Gems/Atom/Feature/Common/Assets/Materials/Types/MaterialInputs/SilhouettePropertyGroup.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "silhouette",
+    "displayName": "Silhouette",
+    "description": "Properties for configuring outline material properties.",
+    "properties": [
+        {
+            "name": "silhouetteType",
+            "displayName": "Silhouette type",
+            "description": "Silhouette, outline or both",
```

Comment:
There is no option for both, according to the enum below the choice is none, silhouette, outline.

Reply:
I'm trying to guess how these were setup based on the `HighlightComponent.cpp` component properties

Reply:
I've changed these options to be AlwaysDraw, Visible, XRay and NeverDraw